cypress/e2e/40-manual-partial-execution.cy.ts

```diff
@@ -22,8 +22,8 @@ describe('Manual partial execution', () => {
 
 		canvas.actions.openNode('Webhook1');
 
-		ndv.getters.nodeRunSuccessIndicator().should('exist');
-		ndv.getters.nodeRunTooltipIndicator().should('exist');
-		ndv.getters.outputRunSelector().should('not.exist'); // single run
+		ndv.getters.nodeRunSuccessIndicator().should('not.exist');
+		ndv.getters.nodeRunTooltipIndicator().should('not.exist');
+		ndv.getters.outputRunSelector().should('not.exist');
```

Comment:
The second webhook should not be executed at all:
![image](https://github.com/user-attachments/assets/b0b9319f-170a-4073-8d62-387976a604ad)